### ISSUE TITLE
[BUG] Can't get value of subfeature in0_input: Can't read

### ISSUE THREAD

**Issue by MuriloucoLouco** [HorlogeSkynet/archey4#100]
**Describe the bug**
When i run archey, it gives the following message:
```
WARNING: [lm-sensors]: ERROR: Can't get value of subfeature in0_input: Can't read
ERROR: Can't get value of subfeature in1_input: Can't read
```
And don't display the temperature correctly (it says `Temperature: 47.3 C (Max. 47.5 C)`).

**Expected behavior**
To show the temperature correctly and not give an error.

**Environment**
 - Version used (4.Y.Z) : v4.12.0
 - Method of installation (distribution package, PyPI, Homebrew, sources) : pip install
 - Hardware type (laptop, server, Raspberry, hyper-visor) : Desktop
 - Python version (3.Y.Z) : 3.9.6
 - Operating system and version : 5.11.22-2-MANJARO
 - Graphical environment name and version : KDE 5.22.3
 - Connectivity (off-line, LAN only, Internet access) : Internet access

**Additional context**
I have all dependencies installed.

This is the output of `sensors -u`:
```
amdgpu-pci-0700
Adapter: PCI adapter
vddgfx:
ERROR: Can't get value of subfeature in0_input: Can't read
vddnb:
ERROR: Can't get value of subfeature in1_input: Can't read
edge:
  temp1_input: 39.000

k10temp-pci-00c3
Adapter: PCI adapter
Tctl:
  temp1_input: 39.125
Tdie:
  temp2_input: 39.125
```

**Comment by HorlogeSkynet** [comment]
Hello !

* About the warning/error message : this happens when `sensors` cannot fetch some feature values (for instance, in my very case, it occurs when the WiFi controller is disabled, as it contains a detected thermal sensor as well). You can hide it by setting the `suppress_warnings` configuration option to `true`. If you don't want to hide warning messages globally, you will want to populate `sensors_chipsets` _white-list_ containing "smoothly-working" chipsets names (in your case `k10temp-pci-00c3`). You can refer to configuration documentation.
* About the supposedly wrong temperature : `Temperature` entry computes an average value based on multiple sources. In your case (GNU/Linux on desktop), detected `sensors` features are being summed with CPU (and maybe others) thermal sensors values (from `/sys/class/thermal/thermal_zone*/temp` files). That's why the value displayed can differ from a simple `sensors` execution.

Bye 👋

**Comment by HorlogeSkynet** [comment]
Closing here, do not hesitate to ask for help or even disapprove my points.

**Comment by HorlogeSkynet** [comment]
@MuriloucoLouco there was actually a bug with the `sensors_chipsets` option when multiple items were specified. This has been fixed in 063b19d and released as [v4.13.0](https://github.com/HorlogeSkynet/archey4/releases/tag/v4.13.0).
